R/sleeper_starterpositions.R

```diff
@@ -26,8 +28,12 @@ ff_starter_positions.sleeper_conn <- function(conn, ...) {
     dplyr::count(name = "min") %>%
     dplyr::ungroup() %>%
     dplyr::mutate(
-      total_starters = sum(.data$min, na.rm = TRUE),
       pos = purrr::map_chr(.data$pos, unlist)
+    ) %>%
+    dplyr::full_join(all_positions, by=dplyr::join_by(pos)) %>%
+    replace(is.na(.),0) %>%
```

Comment:
`dplyr::join_by()` is relatively new syntax, so would prefer to use "pos" here in case users have an older version of dplyr that doesn't support it. 

More broadly could do these two operations in one step via [tidyr::complete()](https://tidyr.tidyverse.org/reference/complete.html):

```suggestion
    tidyr::complete(pos = all_positions$pos, fill = list(min = 0)) %>%
```
(also probably could use just a vector of all_positions rather than a dataframe here)